Makefile

```diff
@@ -12,7 +12,7 @@ BuildDate = $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 GIT_COMMIT ?= $(shell git rev-parse --short HEAD)
 ALL_ARCH = amd64 arm arm64 ppc64le s390x
 PKG = k8s.io/kube-state-metrics/v2/pkg
-GO_VERSION = 1.14.7
+GO_VERSION = 1.14.10
```

Comment:
Would you mind making the change to 1.15.3 all over the repo? Planned on doing the switch myself but since you opened the PR. 

We have a few other places if you are willing to migrate to 1.15 :) 
https://github.com/kubernetes/kube-state-metrics/blob/f1166b40298a6846a329f567d0d509d32ee45a81/.github/workflows/ci.yml

Reply:
Sure, will do!